pkg/aws/eni/node_manager.go

```diff
@@ -162,6 +132,22 @@ func (n *NodeManager) Update(resource *v2.CiliumNode) bool {
 			name:    resource.Name,
 			manager: n,
 		}
+
+		deficitResolver, err := trigger.NewTrigger(trigger.Parameters{
```

Comment:
It looks like the number of API requests is the same, it's just that the triggers are per-node now - is that correct?

Reply:
Correct